llama-index-python-kvstore/run.sh

```diff
@@ -17,6 +17,8 @@ $PYTHON_BINARY -m pip install -U pip poetry
 $PYTHON_BINARY -m poetry env use $PYTHON_BINARY
 # Activate the poetry env, which itself does not include poetry
 source $($PYTHON_BINARY -m poetry env info --path)/bin/activate
+# Fix requirement in llama-index repo
+$PYTHON_BINARY -m poetry add motor
```

Comment:
This instead of closing Shruti's branch and opening up another PR? Or maybe commenting on her PR?

Reply:
I'll add the JIRA ticket, but think this is a good temporary stop-gap since waiting on Shruti's PR for approval is an external dependency.